packages/dds/tree/src/test/snapshots/schema-files/recursiveType-recursive.json

```diff
@@ -91,6 +91,16 @@
         }
       }
     },
+    "test.hasIdentifierField": {
+      "object": {
+        "field": {
+          "kind": "Identifier",
+          "types": [
+            "com.fluidframework.leaf.string"
```

Comment:
is this the long-term serialized schema we want (typed as string)? it seems fine either way, just making sure this is an intentional choice.

Reply:
@noencke @CraigMacomber